bulk_indexer.go

```diff
@@ -59,7 +59,7 @@ type bulkIndexer struct {
 
 // MinimalBulkIndexerResponse represents a minimal subset of the Elasticsearch _bulk API response.
 type MinimalBulkIndexerResponse struct {
-	Items []map[string]BulkIndexerResponseItem `json:"items,omitempty"`
+	Items []BulkIndexerResponseItem `json:"items,omitempty"`
```

Comment:
Should we remove the `MinimalBulkIndexerResponse` type, and just return `[]BulkIndexerResponse` everywhere?

Reply:
Good catch! Updated!